web/app/js/components/Version.jsx

```diff
@@ -51,7 +60,7 @@ class Version extends React.Component {
   render() {
     return (
       <div className="version">
-        Running {this.props.productName || "controller"} {this.props.releaseVersion}.<br />
+        Running {this.props.productName || "controller"} {this.numericVersion(this.props.releaseVersion)}.<br />
```

Comment:
do we want to tell the user what channel they are on? (i realize it can be inferred by the version number)